terraform/testcases/datadog_exporter_metric_mock/parameters.tfvars

```diff
@@ -4,3 +4,5 @@ soaking_data_mode = "metric"
 sample_app = "spark"
 
 sample_app_image = "public.ecr.aws/aws-otel-test/aws-otel-java-spark:latest"
+
+otconfig_args = ["--feature-gates=-adot.exporter.datadogexporter.deprecation"]
```

Comment:
We might not need the flag passed for datadog, as we have the datadag exporter in stagealpha and enabled by default.

Reply:
Yes, but doing it now ensures that when the flag changes there won't be a chance it is forgotten here.

Reply:
yeah Agreed, makes sense